### PR TITLE
Adding aesgcm info back in and mentioning migrating between types

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5977

Link to docs preview:
https://59554--docspreview.netlify.app/openshift-enterprise/latest/security/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/enabling-etcd-encryption.adoc [hunk 1]
```diff
@@ -20,6 +20,13 @@ After you enable etcd encryption, several changes can occur:
 * A disk I/O can affect the node that receives the backup state.
 ====
 
+You can encrypt the etcd database in either AES-GCM or AES-CBC encryption.
+
+[NOTE]
+====
+If you want to migrate your etcd database from one encryption type to the other, you can modify the API server's `spec.encryption.type` field, and the etcd Operator will migrate the encryption type without having to unencrypt the etcd database first.
```
Comment (kcarmichael08) [inline modules/enabling-etcd-encryption.adoc]: suggestion:
```suggestion
To migrate your etcd database from one encryption type to the other, you can modify the API server's `spec.encryption.type` field. The etcd Operator then migrates the encryption type without having to unencrypt the etcd database first.
```

**Review by kcarmichael08 — COMMENTED** [review]
Small optional suggestion; otherwise lgtm!

File: modules/enabling-etcd-encryption.adoc [hunk 2]
```diff
@@ -20,6 +20,13 @@ After you enable etcd encryption, several changes can occur:
 * A disk I/O can affect the node that receives the backup state.
 ====
 
+You can encrypt the etcd database in either AES-GCM or AES-CBC encryption.
+
+[NOTE]
+====
+To migrate your etcd database from one encryption type to the other, you can modify the API server's `spec.encryption.type` field. {product-title} then migrates the encryption type without having to unencrypt the etcd database first.
```
Comment (dinhxuanvu) [inline modules/enabling-etcd-encryption.adoc:27]: I'm not sure what the `{product-title}` will be? I would like to change this sentence to be "Then data encryption migration to the new encryption type will happen automatically" or something along those lines.

**Review by dinhxuanvu — CHANGES_REQUESTED** [review]

File: modules/enabling-etcd-encryption.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/enabling-etcd-encryption.adoc]: Hey @dinhxuanvu if you look at the preview link I shared (https://59554--docspreview.netlify.app/openshift-enterprise/latest/security/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd) you can see how it resolves. {product-title} will turn into "OpenShift Container Platform".

However, I like your suggestion, so I've reworded similar to that. PTAL now and let me know what you think. Thanks!

Note that the auto previews actually just stopped working right now, so I'm uploading a new preview here (VPN required): 

http://file.rdu.redhat.com/~ahoffer/2023/OSDOCS-5977/security/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd

**Review by dinhxuanvu — COMMENTED** [review]
/lgtm

**Comment by bergerhoffer** [comment]
@dpuniaredhat Can you please review?

http://file.rdu.redhat.com/~ahoffer/2023/OSDOCS-5977/security/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd

**Pull request merged**